### ISSUE TITLE
Incompatible type: T should be ...

### ISSUE THREAD

**Issue by CrazyFlasher** [HaxeFoundation/intellij-haxe#973]
Below line highlights as error. How to ignore it?
```
var movie:IMovieClip = factory.getInstance(IMovieClip);
```
```
function getInstance<T>(type:ClassRef<T>, ?name:MappingName, targetType:Class<Dynamic> = null,
                            ignorePool:Bool = false):T;
```

**Comment by EricBishton** [comment]
How about a minimal sample, so that I can duplicate the error?  (Perhaps something that compiles on http://try.haxe.org). Also, what version of the plugin and IDEA are you using?  

Finally, there is a test release out on https://github.com/EricBishton/IntelliJ-Haxe/releases that has a number of these types of errors fixed already. You could try that.

**Comment by CrazyFlasher** [comment]
Using latest available in Intellij IDEA: 1.2 for IDEA 2019.x june 22, 2019
https://try.haxe.org/#BAd03
![image](https://user-images.githubusercontent.com/1607138/70077154-e8d45380-1608-11ea-83f9-74aed7e23a97.png)

**Comment by EricBishton** [comment]
Verified.  Still exists in development branch at 64e8853d.

**Comment by CrazyFlasher** [comment]
Is it possible to turn off temporary checking this error in idea?
Currenlty, I am using cast in this cases :)

**Comment by EricBishton** [comment]
No. It's not behind any feature flag.  You can turn off highlighting of errors, in general, (by setting the color of errors to nothing) but then you don't get *any* errors.

If you're willing to build the plugin yourself, then you could disable that check (In HaxeSemanticAnnotator.java: `TypeTagChecker.check()`).

**Comment by CrazyFlasher** [comment]
Ok. I'll better wait for a fix :)

**Comment by EricBishton** [comment]
Fix was added to PR 972 - at 64e8853d.  I'll merge it in the morning, after CI confirms everything is OK.

**Comment by CrazyFlasher** [comment]
Another case:
https://try.haxe.org/#84F8C
![image](https://user-images.githubusercontent.com/1607138/71022075-52278c80-2108-11ea-8084-8103c6b38b26.png)

**Comment by EricBishton** [comment]
That one is fixed, too.  You can pick up a test build at https://github.com/EricBishton/intellij-haxe/releases/tag/Post1.2TestBuild.2  It will be released with 1.3.
